tests/test_phabricator_client.py

```diff
@@ -16,6 +16,8 @@
 from tests.canned_responses.phabricator.repos import *
 from tests.canned_responses.phabricator.errors import *
 
+pytestmark = pytest.mark.usefixtures('docker_env_vars')
```

Comment:
I believe that if you also move this to `test/utils.py` then we won't have to add it at the top of every test file, every test file can just import the utils file as they already do.